intel_extension_for_transformers/llm/operator/cscr/CMakeLists.txt

```diff
@@ -1,21 +1,19 @@
-#  Copyright (c) 2023 Intel Corporation
-#
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-
+##  Copyright (c) 2023 Intel Corporation
+##
+##  Licensed under the Apache License, Version 2.0 (the "License");
+##  you may not use this file except in compliance with the License.
+##  You may obtain a copy of the License at
+##
+##    http://www.apache.org/licenses/LICENSE-2.0
+##
+##  Unless required by applicable law or agreed to in writing, software
+##  distributed under the License is distributed on an "AS IS" BASIS,
+##  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+##  See the License for the specific language governing permissions and
+##  limitations under the License.
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
-project(weight_only_jblasop LANGUAGES C CXX)
+project(QBits LANGUAGES C CXX)
 
-# set(CMAKE_CXX_FLAGS "-g -O0 -Wno-narrowing")
 set(CMAKE_CXX_FLAGS "-O3")
```

Comment:
no need for hard-code CXX_FLAGS

Reply:
fix in [cfd4f0f](https://github.com/intel/intel-extension-for-transformers/pull/294/commits/cfd4f0fffd411e363082bc9cd457af6770e94660)